### PR TITLE
National report pdf page 2

### PR DESCRIPTION
**Pull request by tinashechiraya**
[Screencast from 26-08-23 11:36:38.webm](https://github.com/kartoza/sawps/assets/70011086/11c2187d-82fd-490e-841a-480d6bf6bc81)
Screenshot from 2023-08-26 11-38-05
Screenshot from 2023-08-26 11-38-21

### CODE REVIEW

**Comment by tinashechiraya** [comment]
Description:

national report page 2 ,contains 2 charts ,
the population pyramids and the line charts.
The species colour and icon are dynamic (stored in db).
Each chart line represents a species data for the years where data is available .
The population pyramids display the ratio of female to male ,adults ,sub adults ,juvenile

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 31 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 28 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 24 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by tinashechiraya** [comment]
Description: the font size has been increased to make it readable on the pdf

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 23 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 23 min

File: django_project/frontend/api_views/national_statistic.py [hunk 1]
```diff
@@ -59,14 +62,11 @@ def get_statistics(self, request):
         total property area available for
         owned species (national).
         """
-        organisation_id = request.session.get(
-            'current_organisation_id'
-        )
+        # organisation_id = request.session.get(
```
Comment (dimasciput) [inline django_project/frontend/api_views/national_statistic.py:65]: please update this

File: django_project/frontend/src/components/PdfReport/generateChartImage.tsx [hunk 2]
```diff
@@ -100,6 +134,17 @@ const GenerateChartImages: React.FC = () => {
                   />
               ))}
               </div>
+              // smaller preview
```
Comment (dimasciput) [inline django_project/frontend/src/components/PdfReport/generateChartImage.tsx:137]: remove if unused

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/frontend/api_views/national_statistic.py [hunk 3]
```diff
@@ -59,14 +62,8 @@ def get_statistics(self, request):
...
         """
-        organisation_id = request.session.get(
-            'current_organisation_id'
-        )
 
-        national_properties = Property.objects.filter(
-            property_type__name='national',
-            organisation=organisation_id
-        )
+        national_properties = Property.objects.filter()
```
Comment (dimasciput) [inline django_project/frontend/api_views/national_statistic.py:66]: @tinashechiraya is this correct? It seems that you haven't added any filters.

Comment (tinashechiraya) [reply django_project/frontend/api_views/national_statistic.py:66]: @dimasciput Yes I talked to Luna she informed me that for a national level the user needs to be able to get data country wide regardless of organisation or property type

File: django_project/frontend/src/components/PdfReport/generateChartImage.tsx [hunk 4]
```diff
@@ -44,12 +47,35 @@ const GenerateChartImages: React.FC = () => {
   
         // retrieve the charts refs as canvas
         const first_page_canvas = await html2canvas(firstPageRefs.current);
+        const second_page_canvas = await html2canvas(secondPageRefs.current);
+        // const third_page_canvas = await html2canvas(thirdPageRefs.current);
   
         // convert the canvas to base64 strings
-        const first_page_charts = first_page_canvas.toDataURL('image/png');
+        // const first_page_charts = first_page_canvas.toDataURL('image/png');
```
Comment (dimasciput) [inline django_project/frontend/src/components/PdfReport/generateChartImage.tsx:54]: please clean up the codes

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 24 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 24 min

**Pull request merged**